sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseIntegrationTests.java

```diff
@@ -86,7 +90,7 @@ public void testPingRequestBody() throws InterruptedException {
             "\\{\"Documents\":null,\"InstrumentationKey\":null,\"Metrics\":null,\"InvariantVersion\":1,\"Timestamp\":\"\\\\/Date\\(\\d+\\)\\\\/\",\"Version\":\"testSdkVersion\",\"StreamId\":\"qpid123\",\"MachineName\":\"machine1\",\"Instance\":\"instance1\",\"RoleName\":null\\}";
```

Comment:
Does this test pass? Asking because I'm noting that the expected invariant version is 1 here, but you changed it to 6 in the code.

Reply:
I guess it would pass, it's just a minor inconsistency that wouldn't affect the outcome of the test.

Reply:
@heyams I don't think ping requests are supposed to have request bodies at all. This particular line wasn't something that Naveen modified, but just something small that caught my attention. Can fix later.

Reply:
changed to invariant version 6 and tests still pass